studio/components/interfaces/Settings/Logs/LogTable.tsx

```diff
@@ -44,36 +54,42 @@ const LogTable = ({ isCustomQuery, data }: Props) => {
     },
   }))
 
-  const logMap: { [id: string]: LogData } = useMemo(() => {
-    return (data || []).reduce((acc: any, d) => {
+  const logMap = useMemo(() => {
+    if (!isChrono) return {} as LogMap
+    const logData = data as LogData[]
+    return logData.reduce((acc: LogMap, d: LogData) => {
       acc[d.id] = d
       return acc
-    }, {})
+    }, {}) as LogMap
   }, [JSON.stringify(data)])
 
-  const strLogMap = JSON.stringify(logMap)
+  const strData = JSON.stringify(data)
```

Comment:
full variable names are preferable. 

```suggestion
  const stringData = JSON.stringify(data)
```

Reply:
done